### ISSUE TITLE
Emote issues

### ISSUE THREAD

**Issue by itsMaru** [SapphireMordred/Sapphire#329]
There are a couple issues with emotes.
Could be looked into, and shouldn't be difficult

- [x] Unchecking Display log message and using an emote only hides the message from selfclient, but still displays message to others.
- [x]  Persistent emotes have issues. They need to be stored in the player's class and then be set properly on the appropriate packets (spawn, etc) as well as should be handled when they're cancelled.
- [x] Identify and handle emotes which need weapons drawn previously to being executed to show correctly to other players.

**Comment by JeidoUran** [comment]
Regarding the persistent emotes issue, it actually happens with all of the emotes. Basically, any emote used will always play fully for other players, whether or not the player doing it moves to cancel it (resulting in the character doing the emote while moving on their screens). With persistent ones, it loops endlessly until the player using it respawns.

An other issue I found is that emotes that uses the equipped weapon (such as Zantetsuken or Victory Pose) will not show for other players if they are started with the weapon sheathed. I don't know at all if it's linked, but the walking animation issue I mentionned some time ago is very similar to that (walking animation showing only for the player using it)

Change Pose also suffers from the same problem, and works only for the player using it.
